raiden_installer/base.py

```diff
@@ -68,12 +65,14 @@ def configuration_data(self):
         }
 
         # If the config is for a demo-env we'll need to add/overwrite some settings
-        if self.settings.client_release_channel == "demo_env":  # noqa
-            base_config.update({"matrix-server": self.settings.matrix_server})  # noqa
-            base_config["routing-mode"] = "pfs"
-            base_config[
-                "pathfinding-service-address"
-            ] = self.settings.pathfinding_service_address  # noqa
+        if self.settings.client_release_channel == "demo_env":
+            base_config.update(
+                {
+                    "matrix-server": self.settings.matrix_server,
+                    "routing-mode": "pfs",
+                    "pathfinding-service-address": self.settings.pathfinding_service_address,
+                }
+            )
```

Comment:
Why do we need this case? Can't we just change the values in `demo_env.toml`?

Reply:
These are the values that get saved in a Raiden configuration file. We need to add them there, because the configuration file should also be usable without running the Wizard.